api/sql/ddl/table.sql

```diff
@@ -4,6 +4,12 @@ create type Apt_type as enum (
     'OnNotice'
 );
 
+CREATE TYPE Periodo AS ENUM (
+    'Noturno',
+    'Diurno',
+    'Indiferente'
+);
+
 create type User_type as enum (
     'Employer',
```

Comment:
`Employer` deve ser trocado para `Employee`. Essa modificação já havia sido comentada anteriormente, acho que esse PR seria um bom momento para incluí-la.

```suggestion
    'Employee',
```